### PR TITLE
ENH: Add option to use pytest arguments

### PR DESCRIPTION
**Pull request by ZryletTC**
Sometimes custom pytest parameters can be helpful, so this allows that.

Questions? Comments? Concerns?

### CODE REVIEW

**Review by hhslepicka — APPROVED** [review]
Looks really good! Nice addition!

**Comment by klauer** [comment]
Did you try this already?

```
$ pytest --help
...

environment variables:
  PYTEST_ADDOPTS           extra command line options
  PYTEST_PLUGINS           comma-separated plugins to load during startup
  PYTEST_DISABLE_PLUGIN_AUTOLOAD set to disable plugin auto-loading
  PYTEST_DEBUG             set to enable debug tracing of pytest's internals
```

**Comment by ZryletTC** [comment]
Oh cool, no I didn't notice that. That'd make things simpler. Thanks @klauer!

**Comment by ZryletTC** [comment]
Well I guess this is a little unnecessary then.

**Comment by klauer** [comment]
Might be good for the template to show that variable so we use it consistently in the future

File: example_python_travis.yml [hunk 1]
```diff
@@ -9,9 +9,13 @@ env:
     - DOCTR_VERSIONS_MENU="1"
     # Dependency files used to build the documentation (space separated)
     - DOCS_REQUIREMENTS="dev-requirements.txt requirements.txt"
+
     # Options to be passed to flake8 for package linting. Usually this is just
     # the package name but you can enable other flake8 options via this config
     - PYTHON_LINT_OPTIONS="my_package"
+    # Options to be passed to pytest for package test. Often, this will be left
+    # blank but can be used to enable custom pytest features
+    - PYTEST_ADDOPTS="-W"
```
Comment (klauer) [inline example_python_travis.yml:18]: Looks good, but probably best to leave it blank for when people copy/paste this:

```suggestion
    - PYTEST_ADDOPTS=""
```

```
$ pytest -W
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: argument -W/--pythonwarnings: expected one argument
```

**Pull request merged**